tools/target_test.py

```diff
@@ -22,17 +31,28 @@ def _test(self, executable):
         self.check_exists(bin_file)
         run([bin_file], quiet=True)
 
+    def test_cargo_test(self):
+        if is_cargo_test():
+            if os.environ["DENO_BUILD_MODE"] == "release":
+                run(["cargo", "test", "--all", "--release"])
```

Comment:
Are you sure this doesn't rebuild part of deno (because it's not done with --locked) ?

Reply:
Added locked.